sklearn/pipeline.py

```diff
@@ -705,9 +792,19 @@ def fit(self, X, y=None):
             This estimator
         """
         self._validate_transformers()
+        all_transformers = [(name, trans, weight) for name, trans, weight in
```

Comment:
This diff seems weird to me. It is not in master. I would be checking that this is not from the previous version before `_BaseComposition`

Reply:
It seems this was done, along with the change in `_fit_one_transform`, for the benefit of the verbosity, so not in master.
[Here](https://github.com/scikit-learn/scikit-learn/pull/8568/commits/a990034907b634cbe446d956b58f36ea19296551#diff-dd32b47cafa79252b520b030724ddda9R816) 's the commit in the original PR.